src/native/fs/fs_napi.cpp

```diff
@@ -1388,16 +1422,22 @@ fs_napi(Napi::Env env, Napi::Object exports)
 {
     auto exports_fs = Napi::Object::New(env);
 
-    const char* env_p = std::getenv("GPFS_DL_PATH");
-    if (env_p != NULL) {
-        LOG("FS::GPFS GPFS_DL_PATH=" << env_p);
+    if(const char* env_p = std::getenv("GPFS_DL_PATH")) {
         uv_lib_t *lib = (uv_lib_t*) malloc(sizeof(uv_lib_t));
+        LOG("FS::GPFS GPFS_DL_PATH=" << env_p);
         if (uv_dlopen(env_p, lib)) {
             PANIC("Error: %s\n" << uv_dlerror(lib));
         }
-        if (uv_dlsym(lib, "gpfs_fcntl", (void **) &dlsym_gpfs_fcntl)) {
```

Comment:
You need to keep gpfs_fnctl because Jacky's next PR counts on it.
Just replicate this uv_dlsym call for linkat instead of replacing it.

Reply:
done